syntax = "proto2";

message Transport {
  required int32 index = 1;
  required string blk_no = 2;
  required int32 press3 = 3;
  required float calc_press2 = 4;
  required int32 press4 = 5;
  required float calc_press1 = 6;
  required float calc_press4 = 7;
  required float calc_press3 = 8;
  required float bf_gps_lon = 9;
  required float gps_lat = 10;
  required float speed = 11;
  required string in_dt = 12;
  required float move_time = 13;
  required string dvc_id = 14;
  required float dsme_lat = 15;
  required int32 press1 = 16; 
  required int32 press2 = 17;
  required int32 work_status = 18;
  required string timestamp = 19;
  required string is_adjust = 20;
  required int32 move_distance = 21;
  required float weight = 22;
  required float dsme_lon = 23;
  required string in_user = 24;
  required int32 eqp_id = 25;
  required int32 blk_get_seq_id = 26;
  required string lot_no = 27;
  required string proj_no  = 28;
  required float gps_lon = 29;
  required int32 seq_id = 30;
  required float bf_gps_lat = 32;
  required string blk_dvc_id = 33;
}